app/src/main/res/layout-sw600dp-port/topic_info_fragment.xml

```diff
@@ -95,8 +98,8 @@
         android:id="@+id/see_more_text_view"
```

Comment:
Considering that this is tablet layout, we should actually not use `see_more`. 
Topic Description can contains 5-12 lines (in mobile view) which means that in tablet it will need much less lines. 
Now, because in tablet we can easily show all these lines in single screen, we don't need `see_more` functionality in tablet layout. The description should be fully visible always.

Reply:
This same change will apply in `#1462`

Reply:
Done